modules/component/src/componentNext.ts

```diff
@@ -98,13 +86,38 @@ export class ComponentNext<P1 extends ComponentProps> {
     )
   }
 
+  /**
+   * Creates a new component with initial state as undefined
+   *
+   * ```typescript
+   * import {ComponentNext} from '@action-land/component'
+   *
+   * const component = ComponentNext.empty // creates new component with initial state as undefined
+   * ```
+   */
   static get empty(): ComponentNext<{
     iState: undefined
     oView: void
   }> {
     return ComponentNext.lift(undefined)
   }
 
+  /**
+   * Adds ability to transform component's state matching action type
+   *
```

Comment:
how do we add information about all the various use-cases for matchR. For example, multiple reducers on a single action?

Either we'd add more examples below, or else redirect them to view the testcases

Reply:
Purpose of doc is to provide basic usage, adding example of all use cases here will only confuse the consumer.

Reply:
Add sentence or link asking reader to check out test-cases for further documentation